Composer/packages/client/src/components/DialogWrapper/index.tsx

```diff
@@ -1,19 +1,27 @@
 // Copyright (c) Microsoft Corporation.
 // Licensed under the MIT License.
 
-import React from 'react';
+import React, { useEffect, useState } from 'react';
 import { Dialog, DialogType, IDialogProps } from 'office-ui-fabric-react/lib/Dialog';
 
-import { styles } from './styles';
+import { styles, DialogTypes } from './styles';
 
 interface DialogWrapperProps extends Pick<IDialogProps, 'onDismiss'> {
   isOpen: boolean;
   title: string;
   subText: string;
+  dialogType: DialogTypes;
 }
 
 export const DialogWrapper: React.FC<DialogWrapperProps> = props => {
-  const { isOpen, onDismiss, title, subText, children } = props;
+  const { isOpen, onDismiss, title, subText, children, dialogType } = props;
+  const [currentStyle, setStyle] = useState(styles[dialogType]);
+
+  useEffect(() => {
+    if (dialogType) {
+      setStyle(styles[dialogType]);
+    }
+  }, [dialogType]);
```

Comment:
Just curious if this state is needed? What does it buy us? Could it just be `const currentStyle = styles[dialogType]`?

Reply:
Doesn't buy us anything now. This can be changed.